pkg/perf/utils.go

```diff
@@ -0,0 +1,98 @@
+package perf
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"path"
+
+	ispec "github.com/opencontainers/image-spec/specs-go/v1"
+	"gopkg.in/resty.v1"
+)
+
+func PushImage(imagePath string, username string, passphrase string,
```

Comment:
There are many types of image push, you should have tests for all of them.

https://github.com/opencontainers/distribution-spec/blob/main/spec.md#push